radio/src/gui/colorlcd/model_select.cpp

```diff
@@ -431,7 +431,7 @@ class CategoryEditPage : public PageTab
 
         // Details
         char cnt[19];
-        snprintf(cnt, sizeof(cnt), "%lu %s", category->size(), STR_MODELS);
```

Comment:
This is now probably producing a warning on 64 bit systems like Simulator, as size_t has 64 bits on such systems.

Reply:
then: `snprintf(cnt, sizeof(cnt), "%lu %s", (long unsigned it)category->size(), STR_MODELS);` instead?